### DISCUSSION TITLE
Jellyseerr setup → where can I find/configure my Email Address?

### DISCUSSION THREAD

**Discussion by ThijmenDam** [seerr-team/seerr#248] [Q&A]
Hi there!

This might be a an obvious question to some, but I have no clue where I can find or configure my Jellyfin Email Address, meaning I cannot proceed past the first step of Jellyseerr setup...

Any help would be appreciated, thanks :-)

**Comment by fallenbagel** [comment]
That email can be anything. Its mainly for smtp purposes. It can literally be nothing@nothing.nothing
